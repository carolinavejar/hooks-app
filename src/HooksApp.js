import React from 'react'

export const HooksApp = () => {
    return (
        <div className="container">
            <h1> Hola carito</h1>
        </div>
    )
}
