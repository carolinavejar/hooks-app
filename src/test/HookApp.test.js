import React from 'react';
import { HooksApp }  from '../HooksApp';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme/build';

// No las toma de setupTests.js , así que las dejaré aquí por ahora
import Enzyme from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { createSerializer } from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });
expect.addSnapshotSerializer( createSerializer ( { mode : 'deep' } ));

describe("Pruebas de HooksApp .... .test.js", ()=> { 
    let wrapper = shallow(<HooksApp />);

    test('debe de mostrar <HooksApp /> correctamente', () => {
        expect (wrapper).toMatchSnapshot();
    });

});

