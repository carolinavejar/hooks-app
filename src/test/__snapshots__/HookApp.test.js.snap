// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de HooksApp .... .test.js debe de mostrar <HooksApp /> correctamente 1`] = `
<div
  className="container"
>
  <h1>
     Hola carito
  </h1>
</div>
`;
